test/fs.test.ts

```diff
@@ -0,0 +1,65 @@
+import 'mocha';
+import * as path from 'path';
+import { expect } from 'chai';
+
+import * as fs from '../src/util/fs';
+
+describe('UTIL: fs', () => {
+
+  const filePath = path.resolve(__dirname, '.tmp/hello.txt');
+  const invalidFilePath = path.resolve(__dirname, '.tmp/fake_file.txt/random');
+  const fileContent = 'hello@123';
+
+  const rejectionCallback = (error: { errno: any; code: any; syscall: any; }) => {
+    expect(error.errno).to.equal(-2);
+    expect(error.code).to.equal('ENOENT');
+    expect(error.syscall).to.equal('open');
+  };
+
+  it('should write content to the filepath', done => {
+    fs.write(filePath, fileContent).then(result => {
+      expect(result).to.equal(undefined);
+      done();
+    });
+  });
+
+  it('should read the content from the filepath', done => {
+    fs.read(filePath).then(res => {
```

Comment:
How do you know this file exists? This is not a deterministic approach. All tests need to be independent and if it requires something you should do and undo it in setup / teardown phases.